Formula/nali.rb

```diff
@@ -0,0 +1,28 @@
+class Nali < Formula
+  desc "Tool for querying IP geographic information and CDN provider"
+  homepage "https://github.com/zu1k/nali"
+  url "https://github.com/zu1k/nali/archive/v0.4.2.tar.gz"
+  sha256 "25842823d61b1c05d8e261c28a2f24232838a1397cbe3b227a2c6288ec451fd6"
+  license "MIT"
+  head "https://github.com/zu1k/nali.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+    (bash_completion/"nali").write `#{bin}/nali completion bash`
+    (fish_completion/"nali.fish").write `#{bin}/nali completion fish`
+    (zsh_completion/"_nali").write `#{bin}/nali completion zsh`
+  end
+
+  def caveats
+    <<~EOS
+      Config will be written to the ~/.nali folder.
+    EOS
+  end
+
+  test do
+    ip = "1.1.1.1"
+    assert_match "#{ip} [美国 APNIC&CloudFlare公共DNS服务器]", shell_output("#{bin}/nali #{ip}")
```

Comment:
Is there any output in English?

Reply:
Some part of the program is in English, like the help text. The actual info text comes from external database, and [the only English database (GeoIP2) supported](https://github.com/zu1k/nali/blob/4cccb663b47ac93c6b37d92e5237d46b227a6234/README_en.md#multilingual-support) requires an account to download from MaxMind website. I guess this does make the program less useful for non-Chinese speaking users out of the box though.

It reads "US **APNIC&CloudFlare** Public **DNS** Server", bold text are in English so it's not like completely unrecognizable.

Reply:
Yeah a comment would be nice.